dashboard-widgets.el

```diff
@@ -710,6 +710,12 @@ String -> bool.
 Argument IMAGE-PATH path to the image."
   (eq 'gif (image-type image-path)))
 
+(defun dashboard--type-is-webp-p (image-path)
```

Comment:
Since `webp` images and `gif` images are going to be animated. I suggest an `--image-animated-p` that tells if the image is a `gif` or a `webp`.

Reply:
@ricardoricho i have implemeted a function to check if image has multiple frame. What do you think about this solution?